trade/api/fameex_api.js

```diff
@@ -0,0 +1,481 @@
+        WEB_BASE = apiServer;
+      }
+
+      if (logger) {
+        log = logger;
+      }
+
+      if (!publicOnly) {
+        config = {
+          apiKey,
+          tradePwd,
+          secret_key: secretKey,
+        };
+      }
+    },
+
+    /**
+     * Get user assets balance
+     * https://fameex-docs.github.io/docs/api/spot/en/#get-wallet-info
+     * @return {Promise<Array>}
+     */
+    getBalances() {
+      return protectedRequest('get', `${versioning.v1}/api/account/wallet`, {});
+    },
+
+    /**
+     * Get a list of orders
+     * https://fameex-docs.github.io/docs/api/spot/en/#get-a-list-of-orders
+     * !POSSIBLE IMPLEMENTATION ERRORS!
+     * !At the moment it is impossible to implement this functional correctly, due to problems on the FameEX side
+     * @param {String} base Transaction currency (uppercase, such as "BTC")
+     * @param {String} quote Denominated currency (uppercase, such as "USDT")
+     * @param {Number[]} orderTypes List of order types 1-limit price 2-market price 3-stop profit stop loss 4-tracking order 5-Maker only
+     * @param {Number} state Order status 7-uncompleted 8-completed 9-completed or partially cancelled
+     * @param {Number} pageNum Pagination, the first few pages (1 <= pageNum)
+     * @param {Number} pageSize Pagination, the number of pages (1 <= pageSize <=500)
+     * @return {Promise<Array>}
+     */
+    getOrders(
+        base,
+        quote,
+        orderTypes,
+        state,
+        pageNum,
+        pageSize,
+    ) {
+      const data = {
+        base,
+        quote,
+        orderTypes,
+        state,
+        pageNum,
+        pageSize,
+      };
+
+      return protectedRequest('post', `${versioning.v1}/api/spot/orderlist`, data);
+    },
+
+    /**
+     * Get order details
+     * https://fameex-docs.github.io/docs/api/spot/en/#get-order-details
+     * @param {String} symbol The name of the currency pair, such as "BTC-USDT"
+     * @param {String} orderId Example: '10918742125338689536'
+     * @return {Promise<Object>}
+     */
+    async getOrderDetails(symbol, orderId) {
+      const data = {
+        symbol,
+        orderId,
+      };
+
+      return protectedRequest('post', `${versioning.v1}/api/spot/orderdetail`, data);
+    },
+
+    /**
+     * Get transaction details
+     * https://fameex-docs.github.io/docs/api/spot/en/#get-transaction-details
+     * !POSSIBLE IMPLEMENTATION ERRORS!
+     * !At the moment it is impossible to implement this functional correctly, due to problems on the FameEX side
+     * @param {String} base Transaction currency (uppercase, such as "BTC")
+     * @param {String} quote Denominated currency (uppercase, such as "USDT")
+     * @param {Number} pageNum Pagination, the first few pages (1 <= pageNum)
+     * @param {Number} pageSize Pagination, the number of pages (1 < pageSize <= 500)
+     * @param {String} orderId Order ID
+     * @return {Promise<Object>}
+     */
+    async getTransactionDetails(
```

Comment:
Do we actually need this?

Reply:
Hm, no, I already removed it from the Trader, but forgot to remove from API

Reply:
fixed